metpy/calc/thermo.py

```diff
@@ -22,6 +22,45 @@
 sat_pressure_0c = 6.112 * units.millibar
 
 
+@exporter.export
+@check_units('[temperature]', '[temperature]')
+def relative_humidity_from_dewpoint(temperature, dewpt):
+    r"""Calculate the relative humidity.
+
+    Uses temperature and dewpoint in celsius to calculate relative
+    humidity using the ratio of vapor pressure to saturation vapor pressures
+    and the Magnus Tetens equation
+
+    Parameters
+    ----------
+    temperature : `pint.Quantity`
+        The temperature
+    dew point : `pint.Quantity`
+        The dew point temperature
+
+    Returns
+    -------
+    `pint.Quantity`
+        The relative humidity
+
+    Notes
+    -----
+    Formula:
+    .. math :: e = \exp{(17.269*\frac{T_{d}-273.16}{T_{d}-35.86}})}
+    .. math :: e_{s} = \exp{(17.269*\frac{T-273.16}{T-35.86}})
+
+    .. math :: RH = \frac{e}{e_{s}} * 100
+
+    """
+    a = 17.269
+    e = 6.1078 * units.mb * np.exp(a * (dewpt - (273.16 * units.kelvin)) /
```

Comment:
Instead of adding more manual calculations, can you instead use the existing calculations we have? It looks like this could be written as:
```python
e = saturation_vapor_pressure(dewpt)
e_s = saturation_vapor_pressure(temperature)
```

Our `saturation_vapor_pressure` function is a slightly different variant of the Magnus formula. By re-using we keep things more easily maintained, and we can update in one place our saturation vapor pressure calculation, and everything else benefits.

You should probably also update the docstrings once you've done this as well.

Reply:
I can definitely use those functions! I don't know how I didn't see them before.